config/foreman-answers.yaml

```diff
@@ -54,7 +54,7 @@ foreman::plugin::snapshot_management: false
 foreman::plugin::statistics: false
 foreman::plugin::tasks: false
 foreman::plugin::templates: false
-foreman_proxy: {}
+foreman_proxy: { dhcp_manage_acls: true }
```

Comment:
This shouldn't be needed if we change the module default (which we did).

Reply:
Right, we just need to release, right?

Reply:
Nightly builds use git master